atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/PostgresGetRange.java

```diff
@@ -138,7 +139,8 @@ public PostgresGetRange(PostgresPrefixedTableNames prefixedTableNames,
                                                long timestamp) {
         int maxRowsPerPage = RangeHelpers.getMaxRowsPerPage(rangeRequest);
         int cellsPerRowEstimate = getCellsPerRowEstimate(tableRef, rangeRequest);
-        int maxCellsPerPage = maxRowsPerPage * cellsPerRowEstimate + 1;
+        int maxCellsPerPage = Math.min(
```

Comment:
We should log something if we pick `MAX_BATCH_SIZE`.
Probably at warn - users likely can't do much about it, as it's probably an upstream product bug, but we would like the issue to be highlighted (probably via our internal log collection product) so that the upstream bug can be fixed.